site/en/workflows.md

```diff
@@ -433,17 +497,64 @@ By now the image should be well prepared for segmentation.
 ### Step 7: Region segmentation
 
 <!-- BEGIN-EVAL sed -n '0,/^## Notes/ p' ./repo/ocrd-website.wiki/Workflow-Guide-region-segmentation.md|sed '$d' -->
-
 In this processing step, an (optimized) document image is taken as an input and the
 image is segmented into the various regions, including columns.
 Segments are also classified, either coarse (text, separator, image, table, ...) or fine-grained (paragraph, marginalia, heading, ...).
 
-**Note:** If you use `ocrd-tesserocr-segment-region`, which uses only bounding boxes instead of polygon coordinates,
-then you should post-process via `ocrd-segment-repair` with `plausibilize=True` to obtain better results without large overlaps.
+**Note:** If you use `ocrd-tesserocr-segment-region`, which uses only bounding
```

Comment:
I agree, but this is all still quite fresh, so we should keep all options in there for the moment. The current ordering makes explanation/description easier, but requires a longer read. Practically, yes, I would recommend single-level segmentation with `shrink_polygons=True` over multi-level segmentation (because of workflow freedom) over singe-level segmentation with external post-processing.

Reply:
Agreed that the change is quite recent (even unmerged in the case of shrink_polygons) but @EEngl52 has a point about users chosing the first thing they see and sticking with it.

Reply:
Sure, go ahead. Whatever formulation you come up with, I'll try to correct if necessary.

Reply:
I stole part of your nice summary :) with `--author bertsky` though, so that makes it okay. 👼

Reply:
Cannot find it, sorry.